core/dbsecurity/auth/src/dbUserAuth.cpp

```diff
@@ -1837,13 +1837,74 @@ static void logAuthenticationErrors(int nodeID)
 // *                                                                           *
 // *****************************************************************************
 static void logAuthenticationOutcome(
-   const string   & external_user_name,
-   const string   & internal_user_name,
-   const int32_t    user_id,
-   ClientContents & clientInfo,
-   const string   & outcome)
+   const string       & external_user_name,
+   const string       & internal_user_name,
+   const int32_t        user_id,
+   ClientContents     & clientInfo,
+   const AUTH_OUTCOME   outcome)
    
 {
+   string internalUser("??");
+   if (user_id > 0) 
+     internalUser = internal_user_name; 
+   logLevel severity = LL_INFO;
+
+   // Currently this code has hard coded error messages in many places, this
+   // need to be fixed in order to allow errors to be reported in different 
+   // languages.
+   string outcomeDesc;
+   switch (outcome)
+   {
+      case AUTH_OK:
+         outcomeDesc = "Authentication successful";
+         severity = LL_INFO;
+         break;
+      case AUTH_NOT_REGISTERED:
+         outcomeDesc = "User not registered";
```

Comment:
I notice that we initialize severity to LL_INFO at line 1850, then in all cases but this one, we set it again (sometimes to LL_INFO and sometimes to something else). In this case because we didn't set it, it remains LL_INFO. Just checking: Is this an oversight? Or is LL_INFO intended here? (In which case the code works fine; it's just stylistically inconsistent.)